### DISCUSSION TITLE
Last purchase

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#619] [General]
Title says it all.
What did you last spend your money on?

**Comment by hernessteve** [comment]
-headphones
-food
-train tickets
-new sneakers

**Comment by MirkosMangle** [comment]
I got myself a new fridge from criterion appliances.
It was about time cause the old one didn't function properly anymore.
First it was kinda tricky to set it up but after contacting their customer service here [https://criterion-appliances.pissedconsumer.com/review.html](https://criterion-appliances.pissedconsumer.com/review.html) the installation process went much faster.
